cmake/example/CMakeLists.txt

```diff
@@ -1,10 +1,21 @@
 cmake_minimum_required(VERSION 2.8)
+
+# options
+set(MCU_BUILD_TYPE small)
+set(MCU_FAMILY f4)
+set(MCU_HSE 8000000)
+set(STM32_version 040100)
+
+
+# actual build
+set(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "")
+set(CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "")
```

Comment:
Can you describe the circumstances which made this necessary? I have never had to do it in my gnuarm projects. Which compiler are you on?